### ISSUE TITLE
"Check Project" Only Scans One File

### ISSUE THREAD

**Issue by grvhi** [leinardi/mypy-pycharm#3]
### Step 1: Are you in the right place?
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [x] I have verified that I am using the latest version of the plugin.

### Step 2: Describe your environment
 - Plugin version: `0.7.0`
 - PyCharm/IDEA version: `PyCharm 2018.2.2`
 - Mypy version: `mypy 0.620`

#### Steps to reproduce:

  1. Click "Check Project"

#### Observed Results:

  * The scan runs, but only one file - which happens to be the first python file in the project's first module - is scanned.
  * Clicking "Check Current File" results in the currently active python file (which is not the same file as that which is scanned when clicking "Check Project") being scanned as expected.

  

#### Expected Results:

  * I expected that the scan would run against all python files - recursively - in all modules underneath the project root, as defined in PyCharm's Project Structure settings

**Comment by leinardi** [comment]
Hi @grvhi, thank you for reporting this issue.
Which violations are you getting as result of the scan? Can you please post a screenshot?
Are you getting some error inside the Event Log? If yes could you post it here?
It would be possible to share a sample project where I can reproduce the issue?

**Comment by grvhi** [comment]
@leinardi - perhaps you could help me by clarifying how the plugin interprets "Project" in this context? Does it use the value of Content Root in the Project Structure preferences panel?

**Comment by leinardi** [comment]
The Scan Project is just getting all the non excluded python files in the entire PyCharm project folder:
https://github.com/leinardi/mypy-pycharm/blob/26fbd15d820b1da3fe43b0b4a19f3325b8a92e21/src/main/java/com/leinardi/pycharm/mypy/actions/ScanEverythingAction.java#L39
https://github.com/leinardi/mypy-pycharm/blob/26fbd15d820b1da3fe43b0b4a19f3325b8a92e21/src/main/java/com/leinardi/pycharm/mypy/util/VfUtil.java#L70

I fear that `mypy` is simply not outputting any more error after the first one.

Could you please do a test for me and fix all the errors you are getting reported to see if the scan will show something different?
If that doesn't bring us anywhere, in 3-4 hours I can provide you a debug build with some more log to get more detail about the scan (but if you can, please, try to the above test and let me know).

**Comment by grvhi** [comment]
Thanks @leinardi - your fear is indeed correct. Fixing that mypy warning I posted a screenshot of above, then re-running the Check Project, yields errors from a different file in the project.

**Comment by leinardi** [comment]
Yeah, it is the current behavior of `mypy`, some specific error just breaks the validation of all the other files.

I have created an issue on their tracker: https://github.com/python/mypy/issues/5564

Blocked by python/mypy#5564

**Comment by leinardi** [comment]
Unfortunately I cannot even find out if the check encountered a blocking error because there errors are currently not marked. There is an issue open in April 2017 about this: python/mypy#3850
